React/Modules/RCTUIManager.m

```diff
@@ -64,6 +64,8 @@ @interface RCTAnimation : NSObject
 
 @end
 
+static int _currentKeyboardAnimationCurve;
```

Comment:
Can you make this `UIViewAnimationCurve` instead of `int`? I know that the value it's set to doesn't actually appear in the `UIViewAnimationCurve` enum, but it's intended to be passed to a method parameter of that type so arguably it's a private member of that enum.